routing/localchans/manager.go

```diff
@@ -182,9 +192,14 @@ func (r *Manager) updateEdge(tx kvdb.RTx, chanPoint wire.OutPoint,
 		newSchema.FeeRate,
 	)
 
-	inboundFee := newSchema.InboundFee.ToWire()
-	if err := edge.ExtraOpaqueData.PackRecords(&inboundFee); err != nil {
```

Comment:
Do we know what else has been saved in this `ExtraOpaqueData` field? I think `PackRecords` will overwrite it - if we have other fields they will be lost.

Reply:
There probably should be an `UpdateRecords` method on `ExtraOpaqueData`, it first reads the existing records, then updates the specified records.

Reply:
I think we should just lift the field directly into the struct, rather than making a callers aways encode it and decode it. Not sure the rational of implementing it like this in the first place... 

If we just set the field, then we also ensure that nothing else gets clobbered.

Reply:
I can add the fields to the edge.